src/ApparentHorizons/Tags.hpp

```diff
@@ -202,6 +203,26 @@ struct NormalOneForm : db::ComputeTag {
                        const aliases::OneForm<Frame>& r_hat) noexcept;
   using argument_tags = tmpl::list<DxRadius<Frame>, Rhat<Frame>>;
 };
+/// The OneOverOneFormMargnitude is the reciprocal of the magnitude of the
+/// one-form perpendicular to the horizon
+struct OneOverOneFormMagnitude : db::SimpleTag {
+  using type = DataVector;
+};
+/// Computes the reciprocal of the magnitude of the one form perpendicular to
+/// the horizon
+template <size_t Dim, typename Frame, typename DataType>
+struct OneOverOneFormMagnitudeCompute : db::ComputeTag,
+                                        OneOverOneFormMagnitude {
+  using base = OneOverOneFormMagnitude;
+  static DataVector function(
+      const tnsr::II<DataType, Dim, Frame>& inverse_spatial_metric,
+      const tnsr::i<DataType, Dim, Frame>& normal_one_form) noexcept {
+    return 1.0 / get(magnitude(normal_one_form, inverse_spatial_metric));
```

Comment:
here you will need to say `*one_over_magnitude = 1.0 ...` instead of returning something